extensions/schema-compare/src/schemaCompareMainWindow.ts

```diff
@@ -844,7 +844,12 @@ export class SchemaCompareMainWindow {
 					case mssql.SchemaCompareEndpointType.Database:
 						result = await service.schemaComparePublishDatabaseChanges(this.comparisonResult.operationId, this.targetEndpointInfo.serverName, this.targetEndpointInfo.databaseName, azdata.TaskExecutionMode.execute);
 						break;
-					case mssql.SchemaCompareEndpointType.Project: // Project apply needs sql-database-projects updates in (circular dependency; coming next) // TODO: re-add this and show project logic below
+					case mssql.SchemaCompareEndpointType.Project: // Project apply needs sql-database-projects updates in (circular dependency; coming next)
+						result = await vscode.commands.executeCommand(loc.sqlDatabaseProjectsPublishChanges, this.comparisonResult.operationId, this.targetEndpointInfo.projectFilePath, this.targetEndpointInfo.folderStructure);
+						if (!result.success) {
+							await vscode.window.showErrorMessage(loc.applyError);
```

Comment:
this can probably be void instead of await

Reply:
Specially it *should* be since otherwise you'll be waiting here forever until the user dismisses the message (same for the other ones pointed out)

Reply:
Good catch